conda_build/verify.py

```diff
@@ -0,0 +1,41 @@
+from os.path import join
+import pkgutil
+from .config import context
+from .exceptions import VerifyError
+
+
+def list_verify_script(path_to_script):
+    verify_modules = pkgutil.iter_modules([path_to_script])
+    files = []
+    for _, name, _ in verify_modules:
+        files.append(name)
+    return files
+
+
+def verify(verify_type, ignore_scripts, **kargs):
+    import_type = "verify.%s" % verify_type
+    try:
+        verify_module = __import__(import_type)
+    except ImportError as e:
+        print("can't find verify.recipe module, skipping verification")
```

Comment:
might need a different name for the other project than "verify" - but we'll see.